src/index.ts

```diff
@@ -569,35 +576,50 @@ class ForkTsCheckerWebpackPlugin {
   }
 
   private spawnService() {
+    const env: { [key: string]: string | undefined } = {
+      ...process.env,
+      TYPESCRIPT_PATH: this.typescriptPath,
+      TSCONFIG: this.tsconfigPath,
+      COMPILER_OPTIONS: JSON.stringify(this.compilerOptions),
+      TSLINT: this.tslintPath || (this.tslint ? 'true' : ''),
+      CONTEXT: this.compiler.options.context,
+      TSLINTAUTOFIX: String(this.tslintAutoFix),
+      WATCH: this.isWatching ? this.watchPaths.join('|') : '',
+      WORK_DIVISION: String(Math.max(1, this.workersNumber)),
```

Comment:
I don't know if this is an issue or not but I just noticed that the options now generally use `String(....)` for values. So what we're previously `number` or `boolean` will now be `string`....

Possible issue?